netctl/netctl.go

```diff
@@ -527,14 +530,22 @@ func createNetwork(ctx *cli.Context) {
 	}
 	if gatewayv6 != "" {
 		if ok := net.ParseIP(gatewayv6); ok == nil {
-			errExit(ctx, exitHelp, "Invalid IPv6 gateway ", true)
+			errExit(ctx, exitHelp, "Invalid IPv6 gateway", true)
+		}
+	}
+
+	var pktTag int
+	var err error
+	if len(pktTagInString) > 0 {
+		pktTag, err = strconv.Atoi(pktTagInString)
```

Comment:
nit, drop the declaration of err on 538 and use
pktTag, err := strconv.Atoi(pktTagInString)

Reply:
I can't as pktTag need to be used in line 560.

Reply:
just the err, pktTag still decalred on 537, not a blocker